pkg/salt-minion.sleep

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 SUSE LINUX GmbH, Nuernberg, Germany.
+
+if [ "$1" = pre ] ; then
+	systemctl stop salt-minion
+fi
+if [ "$1" = post ] ; then
+	systemctl start salt-minion
```

Comment:
just noticed that we probably only want to start it when it was running before the suspend... will try to update it tomorrow.